src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-function/GeneFunction.tsx

```diff
@@ -56,7 +58,7 @@ const GeneFunction = (props: Props) => {
         body: styles.panelBody
       }}
     >
-      <div>Panel content is coming...</div>
+      <ProteinsList geneId={props.geneId} />
```

Comment:
Do you want to show the proteins list only when the Proteins tab is active?

Reply:
✔️